run_test.sh

```diff
@@ -121,12 +122,13 @@ function run_webdriver_test {
     echo -e "#### Running webdriver tests in parallel"
     python3 -m pytest --tests-per-worker auto webdriver_tests/*.py
   else
-    python3 -m pytest webdriver_tests/*.py
+    python3 -m pytest webdriver_tests/ranking_test.py
```

Comment:
change this back to `webdriver_tests/*.py`?

Reply:
oops! done.